docs/src/einsum.rst

```diff
@@ -0,0 +1,5 @@
+Einsum Ops
+===============
+
+.. doxygengroup:: einsum
```

Comment:
I'm not sure if this is auto generated. But if not, I would probably just put `einsum` in the `ops.rst`.. it seems a bit overkill to have it's own file and header for a single function.